test/EFCore.MySql.IntegrationTests/Tests/Attributes/SkipJsonFactAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Xunit;
+
+namespace Pomelo.EntityFrameworkCore.MySql.IntegrationTests.Tests.Attributes
+{
+    public sealed class SkipJsonFactAttribute : FactAttribute
+    {
+        public SkipJsonFactAttribute()
+        {
+            if (!AppConfig.ServerVersion.SupportsJson)
+            {
+                Skip = "Version of MySQL/MariaDB does not support JSON";
+            }
+        }
+    }
+}
```

Comment:
This can be removed, if the `SupportedServerVersionFactAttribute` is used instead.

Reply:
Ported `SupportedServerVersionFactAttribute` to `Pomelo.EntityFrameworkCore.MySql.IntegrationTests`